src/core/components/table/TableList.vue

```diff
@@ -65,6 +68,16 @@ export default {
     update (event) {
       this.$emit('update:pagination', event);
     },
+    goTo (row) {
+      if (!this.path) return {};
```

Comment:
Tu pourrais pas retourner directement {} ici si row._id n'est pas défini ?
Ça éviterait de faire 2 conditions

Reply:
row et path ne sont corrélé, ce n'est pas parce qu'il n'y a pas de path qu'il n'y a pas de row. Dans le cas où on a pas row (TeamDirectory), on a effectivement pas row._id mais on a row et c'est plus un hasard que row._id ne soit pas défini donc je ne lierais pas les deux perso

Reply:
Je ne comprends pas ta reponse, comme tu retournes dans les 2 conditions je ne vois pas pourquoi on ne pourrait pas fusionner les 2.

Si tu fais la condition suivante: `if (!this.path || !row._id) return {};`, si il n'y a pas de path ou pas de row._id, on retourne dans les 2 cas `{}`

Reply:
aaah pardon je n'avais pas compris ton retour, je pensais que tu voulais supprimer la condition sur path, je corrige ça